docs/internal-developers/testing/releases/960.md

```diff
@@ -0,0 +1,180 @@
+# Testing notes and ZIP for release 9.6.0
+
+Zip file for testing: [woocommerce-gutenberg-products-block.zip](https://github.com/woocommerce/woocommerce-blocks/files/10730345/woocommerce-gutenberg-products-block.zip)
+
+## WooCommerce Core
+
+### Improved default headings and styling of the cart block and fixed the display of cart and checkout block editable fields when using dark themes. ([8380](https://github.com/woocommerce/woocommerce-blocks/pull/8380))
+
+1. Edit the cart page
+2. Remove the cart block
+3. Insert the cart block
+4. Switch to the "empty cart" view. Confirm the icon is centered.
+5. In the cart block, confirm the "cart totals" text (editable field) is displayed correctly. If you test with a dark theme, it should be transparent with visible text.
+6. Confirm the cross sells are displayed correctly. The title should be a h2 element.
+
+#### Empty Cart Icons
+
+<table>
+<tr>
+<td valign="top">Before:
+<br><br>
+<img alt="image" src="https://user-images.githubusercontent.com/90977/216625777-d2714b91-a3f1-427b-be9f-10cb17132f9b.png">
+</td>
+<td valign="top">After:
+<br><br>
+<img alt="image" src="https://user-images.githubusercontent.com/90977/216625833-e45ec4ee-d4f5-4a14-b67c-0791575c1617.png">
+</td>
+</tr>
+</table>
+
+#### Editable field styling
+
+<table>
+<tr>
+<td valign="top">Before:
+<br><br>
+<img alt="image" src="https://user-images.githubusercontent.com/90977/216626027-3aa7aae4-ca67-4492-acf0-9fb1a8c4e799.png">
+</td>
+<td valign="top">After:
+<br><br>
+<img alt="image" src="https://user-images.githubusercontent.com/90977/216626027-3aa7aae4-ca67-4492-acf0-9fb1a8c4e799.png">
+</td>
+</tr>
+</table>
+
+### Add a reset button for the Filter blocks. ([8366](https://github.com/woocommerce/woocommerce-blocks/pull/8366))
+
+1. Checkout this branch. Run npm run build.
+2. Go to your wp-admin dashboard and create a new page. Add the `All products` block.
+3. Then add each of these blocks `Filter by Price`, `Filter by Stock` and `Filter by Rating`.
+4. Ensure you see the Reset button at the bottom right.
+![Reset Button](https://user-images.githubusercontent.com/2132595/216156231-248e509b-33fb-42cb-bd95-5421ade629d1.png)
+6. Go to the frontend and ensure you see the Reset button after interacting with the block ( check the checkboxes or move the price slider).
+
+### Update the incompatible gateways notice design. ([8365](https://github.com/woocommerce/woocommerce-blocks/pull/8365))
+
+| Before | After |
+| ------ | ----- |
+|<img width="287" alt="image" src="https://user-images.githubusercontent.com/14235870/213469055-ebf0e16f-46e1-4c65-a3da-81739127ba41.png">|![image](https://user-images.githubusercontent.com/14235870/213468877-e956e144-4e9a-48d0-a866-8339ea73e52d.png)|
```

Comment:
Maybe set a width on the second image so it is the same/similar size as the first one?